### ISSUE TITLE
Problems with the Recent apps system

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#254]
```
In the recent apps system, there are two accessibility problems I have noticed, 
which, if resolved, could make the system far more intuitive to use:
   *   When swiping away apps, if there less than a page of apps to swipe away, Talkback doesn't announce the name of the one which has focus.
   *   When swiping away apps, if the swipe action occures in a place not on top of the app, you end up removing a different app.

What version of the product are you using? On what operating system?
Android 4.2.2, Talkback 3.3.2

Please provide any additional information below.
Phone: Nexus4 w/ stock Android Jellybean.

```

Original issue reported on code.google.com by `thomasc...@gmail.com` on 30 Apr 2013 at 11:13

**Comment by GoogleCodeExporter** [comment]
```
By design, two-finger gestures always interact with whatever content is 
directly below the touch point. In recent apps, touch an app to place focus and 
hear the app name, then use two fingers to swipe away the app.
```

Original comment by `alanv@google.com` on 30 Apr 2013 at 7:42
- Changed state: **WontFix**
